docs/3.0.0-beta.x/concepts/configurations.md

```diff
@@ -183,6 +183,25 @@ module.exports = ({ env }) => ({
 | `admin.build`            | Admin panel build configuration                                                                                                                             | Object        |             |
 | `admin.build.backend`    | URL that the admin panel and plugins will request                                                                                                           | string        |             |
 
+## API
+
+**Path —** `./config/api.js`.
+
+```js
+module.exports = {
+  responses: {
+    privateFields: ['_v', 'id', 'created_at'],
```

Comment:
can you rename this to privateAttribues to be consistent :)

Reply:
that is actually a mistake, the code looks for privateAttributes. 
before pushing this change, should I rebase the changes coming, form master?

Reply:
Your PR looks fine now, so not necessary ;)